bower.json

```diff
@@ -15,7 +15,7 @@
   ],
   "license": "BSD-3-Clause",
   "dependencies": {
-    "polymer": "Polymer/polymer#1.0.9",
+    "polymer": "Polymer/polymer#1.1.2",
```

Comment:
the automated tests were failing because other deps aren't pinned:

https://travis-ci.org/webrtc/testrtc/jobs/82963114